build/cmake/build.sh

```diff
@@ -0,0 +1,236 @@
+#!env bash
```

Comment:
All other scripts in the repo seem to invoke `env` as `/usr/bin/env`. Let's be consistent with that here